client/dataset_definitions.go

```diff
@@ -98,6 +98,8 @@ func (s *datasetDefinitions) Delete(ctx context.Context, dataset string) error {
 	if definition.DurationMs.Name == "" {
 		// valid
 	}
+	var dd DatasetDefinition
+	err := s.client.performRequest(ctx, "PATCH", fmt.Sprintf("/1/dataset_definitions/%s", urlEncodeDataset(dataset)), definition, &dd)
```

Comment:
ahh, good catch for the actual write to honeycomb - I don't think I was ever resetting this value after delete externally